app/services/idv/steps/verify_base_step.rb

```diff
@@ -1,10 +1,8 @@
 module Idv
   module Steps
     class VerifyBaseStep < DocAuthBaseStep
-      AAMVA_SUPPORTED_JURISDICTIONS = %w[
-        AR AZ CO CT DC DE FL GA IA ID IL IN KS KY MA MD ME MI MO MS MT NC ND NE
-        NJ NM OH OR PA RI SC SD TN TX VA VT WA WI WY
-      ].to_set.freeze
+      AAMVA_SUPPORTED_JURISDICTIONS = IdentityConfig.store.aamva_supported_jurisdictions.
+      to_set.freeze
 
```

Comment:
I think we ought to just drop the constant now and update [the reference below](https://github.com/18F/identity-idp/blob/5a1cc69b235b1d03131105c332c9dea1fcf10680/app/services/idv/steps/verify_base_step.rb#L79) to use the config value directly.

```suggestion
```

```rb
      def aamva_state?(pii_from_doc)
        IdentityConfig.store.aamva_supported_jurisdictions.include?(pii_from_doc['state_id_jurisdiction'])
      end
```